packages/lib/shared/components/alerts/BalAlertContent.tsx

```diff
@@ -66,7 +66,7 @@ export function BalAlertContent({
             )}
           </HStack>
           {description && (
-            <Text color="black" lineHeight="shorter">
+            <Text as="div" color="black" lineHeight="shorter">
```

Comment:
this shouldn't be necessary for 'text' descriptions

Reply:
Even though is inside a `Text` component, description can also be any `ReactNode` not only just text. I suppose having it inside a text component helps ensure that certain text properties are always set. If rendered as a `<p>` (the default for text components, there are errors  on the console when adding other HTML elements.